packages/amplify-graphql-searchable-transformer/src/generate-resolver-vtl.ts

```diff
@@ -83,15 +96,43 @@ export function responseTemplate(includeVersion = false) {
   return print(
     compoundExpression([
       set(ref('es_items'), list([])),
+      set(ref('aggregateValues'), list([])),
       forEach(ref('entry'), ref('context.result.hits.hits'), [
         iff(raw('!$foreach.hasNext'), set(ref('nextToken'), ref('util.base64Encode($util.toJson($entry.sort))'))),
         ...getSourceMapper(includeVersion),
       ]),
+      forEach(ref('aggItem'), ref('context.result.aggregations.keySet()'), [
+        set(ref('aggResult'), obj({})),
+        set(ref('aggResultValue'), obj({})),
+        qref('$aggResult.put("name", $aggItem)'),
+        iff(
+          raw('!$util.isNullOrEmpty($context.result.aggregations)'),
+          compoundExpression([
+            iff(
+              raw('!$util.isNullOrEmpty($context.result.aggregations.get($aggItem).buckets)'),
+              compoundExpression([
+                qref('$aggResultValue.put("__typename", "SearchableAggregateBucketResult")'),
+                qref('$aggResultValue.put("buckets", $context.result.aggregations.get($aggItem).buckets)'),
+              ]),
+            ),
+            iff(
+              raw('!$util.isNullOrEmpty($context.result.aggregations.get($aggItem).value)'),
+              compoundExpression([
+                qref('$aggResultValue.put("__typename", "SearchableAggregateScalarResult")'),
+                qref('$aggResultValue.put("value", $context.result.aggregations.get($aggItem).value)'),
+              ]),
+            ),
+          ]),
+        ),
+        qref('$aggResult.put("result", $aggResultValue)'),
+        qref('$aggregateValues.add($aggResult)'),
+      ]),
       toJson(
         obj({
           items: ref('es_items'),
-          total: ref('ctx.result.hits.total'),
+          total: ref('ctx.result.hits.total.value'),
```

Comment:
Should it only be `ctx.result.hits.total.value` when using an aggregate search?

Reply:
No, it is not because of aggregate search. It looks like it changed in the latest open search update.